StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/SavedPaymentMethodCollectionView.swift

```diff
@@ -91,11 +103,30 @@ extension SavedPaymentMethodCollectionView {
             button.accessibilityLabel = String.Localized.edit
             return button
         }()
+        lazy var defaultBadge: UILabel = {
+            let label = UILabel()
+            label.font = appearance.scaledFont(for: appearance.font.base.medium, style: .caption1, maximumPointSize: 20)
+            label.textColor = appearance.colors.textSecondary
+            label.adjustsFontForContentSizeCategory = true
+            label.text = String.Localized.default_text
+            label.isHidden = true
+            return label
+        }()
 
         fileprivate var viewModel: SavedPaymentOptionsViewController.Selection?
 
         var isRemovingPaymentMethods: Bool = false {
             didSet {
+                if needsVerticalPaddingForBadge {
```

Comment:
Wdyt: I think the code could be written more linearly:
```
        var isRemovingPaymentMethods: Bool = false {
            didSet {
                updateVerticalConstraintsIfNeeded()
                update()
            }
        }

        func updateVerticalConstraintsIfNeeded() {
            guard needsVerticalPaddingForBadge else {
                return
            }
            if isRemovingPaymentMethods {
                activateDefaultBadgeConstraints()
                defaultBadge.setHiddenIfNecessary(!showDefaultPMBadge)
            } else {
                deactivateDefaultBadgeConstraints()
                defaultBadge.setHiddenIfNecessary(true)
            }
        }
```
This way, when reading `didSet` I don't have to read the implementation of how verticalConstraints get updated-- i can read that constraints get updated if needed, and then a call to update happens irrespective of updating constraints.

Reply:
Yeah, I think that makes sense